ocw/models.py

```diff
@@ -73,8 +93,13 @@ class CspInfo(models.Model):
     type = models.CharField(max_length=200)
 
     def get_openqa_job_link(self):
-        if self.get_tag('openqa_created_by') == 'openqa-suse-de' and self.get_tag('openqa_var_JOB_ID') is not None:
-            url = f"{PCWConfig.get_feature_property('webui', 'openqa_url')}/t{self.get_tag('openqa_var_JOB_ID')}"
+        try:
+            server = get_url(self.get_tag('openqa_var_server'))
```

Comment:
what if get_tag will return None here ?

Reply:
> what if get_tag will return None here ?

It won't pass the `if server and job_id` check later.

Reply:
Fixed.